autoendpoint/src/error.rs

```diff
@@ -174,40 +194,23 @@ impl From<ApiError> for HttpResponse {
 
 impl ResponseError for ApiError {
     fn error_response(&self) -> HttpResponse {
-        // To return a descriptive error response, this would work. We do not
-        // unfortunately do that so that we can retain Sync 1.1 backwards compatibility
-        // as the Python one does.
-        // HttpResponse::build(self.status).json(self)
-        //
-        // So instead we translate our error to a backwards compatible one
-        HttpResponse::build(self.kind.status())
-            .header("Retry-After", RETRY_AFTER.to_string())
-            .finish()
+        HttpResponse::build(self.kind.status()).json(self)
```

Comment:
Why are we not using `Retry-After`?
Should it be an option for `error_response`?

Reply:
The Python code does not use `Retry-After`. This is an artifact of the template taken from sync-storage ([only set if the error is `DbErrorKind::Conflict`](https://github.com/mozilla-services/syncstorage-rs/blob/b2d9341824d3bb7b722e75a5aaaa2e4096007e20/src/error.rs#L226-L228)).

Reply:
Ah! Sorry, you're right.